zowe-rest-api-sample-spring/docs/zos-security.md

```diff
@@ -390,3 +392,57 @@ RECKEY BPX ADD(SERVER UID(userid) SERVICE(READ) ALLOW)
 RECKEY BPX ADD(DAEMON UID(userid) SERVICE(UPDATE) ALLOW)
 F ACF2,REBUILD(FAC)
 ```
+
+## PassTicket Support
+
+It is possible to enable usage of [PassTickets](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.4.0/com.ibm.zos.v2r4.icha700/secsign.htm) for authentication.
+
+The PassTicket is used instead of the password in HTTP Basic Authorization header.
+
+The replay protection needs to be switched off if you need to use the PassTicket multiple times which you need in case of REST API service.
+
+The API service needs to be configured with the APPLID.
```

Comment:
for APPLID, my mind goes to sessions defined in TPX - what is this APPLID?

Reply:
I have added this to the doc:

> The APPLID is a name of the application (up to 8 characters)
> that is used by security products to differentiate certain security operations (like PassTickets) between applications.